sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/NonAzureOpenAISyncClientTest.java

```diff
@@ -396,6 +398,66 @@ public void testGetAudioTranscriptionVtt(HttpClient httpClient, OpenAIServiceVer
         });
     }
 
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.openai.TestUtils#getTestParameters")
+    @RecordWithoutRequestBody
+    public void testAudioTranscriptionTimestampGranularityInWord(HttpClient httpClient, OpenAIServiceVersion serviceVersion) {
+        client = getNonAzureOpenAISyncClient(httpClient);
+
+        getAudioTranscriptionRunnerForNonAzure((modelId, transcriptionOptions) -> {
+            transcriptionOptions
+                    .setResponseFormat(AudioTranscriptionFormat.VERBOSE_JSON)
+                    .setTimestampGranularities(Arrays.asList(AudioTranscriptionTimestampGranularity.WORD));
+
+            AudioTranscription transcription = client.getAudioTranscription(modelId, transcriptionOptions.getFilename(),
+                    transcriptionOptions);
+
+            assertNull(transcription.getSegments());
+            assertAudioTranscriptionWords(transcription.getWords());
+        });
+    }
+
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.openai.TestUtils#getTestParameters")
+    @RecordWithoutRequestBody
+    public void testAudioTranscriptionTimestampGranularityInSegment(HttpClient httpClient, OpenAIServiceVersion serviceVersion) {
+        client = getNonAzureOpenAISyncClient(httpClient);
+
+        getAudioTranscriptionRunnerForNonAzure((modelId, transcriptionOptions) -> {
+            transcriptionOptions
+                    .setResponseFormat(AudioTranscriptionFormat.VERBOSE_JSON)
+                    .setTimestampGranularities(Arrays.asList(AudioTranscriptionTimestampGranularity.SEGMENT));
+
+            AudioTranscription transcription = client.getAudioTranscription(modelId, transcriptionOptions.getFilename(),
+                    transcriptionOptions);
+
+            assertAudioTranscriptionSegments(transcription.getSegments());
```

Comment:
should we assert not empty too?

Reply:
we did it inside of [assertAudioTranscriptionSegements](https://github.com/Azure/azure-sdk-for-java/pull/39660/files#diff-98ebf74b657b3f63b0c36d212b741da5adec3261dcc6fa6666a4d482dd6595a6R699) by using azure core util method. 
`assertFalse(CoreUtils.isNullOrEmpty(segments))`